Exiled.Events/Patches/Events/Player/ChangingWarheadLeverStatus.cs

```diff
@@ -0,0 +1,76 @@
+// -----------------------------------------------------------------------
+// <copyright file="ChangingWarheadLeverStatus.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Events.Player
+{
+#pragma warning disable SA1118
+    using System.Collections.Generic;
+    using System.Reflection.Emit;
+
+    using Exiled.API.Features;
+    using Exiled.Events.EventArgs;
+
+    using HarmonyLib;
+
+    using NorthwoodLib.Pools;
+
+    using static HarmonyLib.AccessTools;
+
+    /// <summary>
+    /// Patches <see cref="PlayerInteract.CallCmdUsePanel"/>.
+    /// Adds the <see cref="Handlers.Player.ChangingWarheadLeverStatus"/> event.
+    /// </summary>
+    [HarmonyPatch(typeof(PlayerInteract), nameof(PlayerInteract.CallCmdUsePanel))]
+    internal class ChangingWarheadLeverStatus
```

Comment:
```suggestion
    internal static class ChangingWarheadLeverStatus
```
All patches must be in a static class

Reply:
Fixed